exchange/utils_test.go

```diff
@@ -3423,6 +3423,163 @@ func TestCleanOpenRTBRequestsBidAdjustment(t *testing.T) {
 	}
 }
 
+func TestCleanOpenRTBRequestsBuyerUID(t *testing.T) {
+	tcf2Consent := "COzTVhaOzTVhaGvAAAENAiCIAP_AAH_AAAAAAEEUACCKAAA"
+
+	buyerUIDAppnexus := `{"appnexus": "a123"}`
```

Comment:
For simplicity in reading the tests, recommend changing a123 to appnexusBuyerId and pubmatic to pubmaticBuyerId